.github/workflows/dev-ci.yml

```diff
@@ -14,21 +14,21 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+        uses: actions/checkout@v4
```

Comment:
@v4 uses the latest v4 version if I understand it well? that's quite handy. But we've been asked for security reasons to pin down the version used with the hash number instead of the version number. See [this article](https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash) for more information about how it could be exploited.